scripts/hostcfgd

```diff
@@ -1108,13 +1115,26 @@ class SshServer(object):
                 if not self.handle_ports_set(value):
                     syslog.syslog(syslog.LOG_ERR, "Failed to update sshd config files - wrong port configuration")
                     return
-            elif int(value) < SSH_MIN_VALUES.get(key, 65535) or SSH_MAX_VALUES.get(key, -1) < int(value):
+                continue
+
+            if key in SSH_INT_VALUES and (int(value) < SSH_MIN_VALUES.get(key, 65535) or
+                                            SSH_MAX_VALUES.get(key, -1) < int(value)):
                 syslog.syslog(syslog.LOG_ERR, "Ssh {} {} out of range".format(key, value))
-            elif key in SSH_CONFIG_NAMES:
+
+            if SSH_CONFIG_NAMES.get(key) is not None:
                 # search replace configuration - if not in config file - append
                 if key == "inactivity_timeout":
                     # translate min to sec.
                     value = int(value) * 60
+                elif key == "password_authentication":
```

Comment:
In the yang model, this field is bool type, seems not necessary to handle string type here.